md/profileCreation.md

```diff
@@ -2,11 +2,23 @@
 
 ## Definition
 
-Basically, a **profile** is an XML file. It contains the profile name, a description, the mapping with the organization (which users / groups / roles / memberships have this profile) and the Bonita Portal menu entries (menus accessible to a user logged in the portal with this profile).  
-A profile must be imported onto the portal, in the **Profiles** tab, to enable the profile (you need to connect with the **Administrator** profile to do it).  
-Since Bonita 7.6.0, Bonita Studio offers this feature, for better packaging and deployment.
-For development purposes, the Studio can **deploy** profiles onto the portal for you (_only for Performance and Efficiency editions_).
-
+### Fonctional
+Profiles are used to grant some rights to some entities of the organization (users / groups / roles / memberships).
+Those rights can be:
+- Accessing a Portal menu
+- Accessing a Living Application content (introduced in Bonita 7.0.0)
+- Accessing business data (through BDM access control) (introduced in Bonita 7.7.0)
+
+A profile is made of a name, a description, and a mapping with the organization entities. 
+When the profile is used to access a Portal menu, Portal menu entries must be defined in the profile too.  
+(When used to access a Living Application, the application menu is defined in the [application descriptor](applicationCreation.md) itself.)  
+Since Bonita 7.6.0, profiles can be managed in the studio, for easier packaging and deployment.  
```

Comment:
I think we should not highlight when the change happen because when page will be the same for version 7.8 for example information will seems to be outdated

Reply:
ok